package.json

```diff
@@ -7,6 +7,7 @@
     "test": "tests"
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
By the way: it look like were not affected by the pullback of node packaged which occurred today:
http://www.heise.de/newsticker/meldung/JavaScript-Paket-aus-NPM-entfernt-Node-Babel-und-Co-scheiterten-beim-Build-3148796.html